### DISCUSSION TITLE
Unattended Access only for logged out sessions (Windows)

### DISCUSSION THREAD

**Discussion by lionking102** [rustdesk/rustdesk#9269] [Q&A]
Hi,

I just have a quick question: Is it possible in Rustdesk to connect to logged out windows sessions without any confirmation, but still have the confirmation window as soon as someone is logged in?
I'm not only referring to the open-source version of Rustdesk, but I would also like to know if it is generally possible, perhaps even with the Server-Pro.

Thanks for your time.

**Comment by rustdesk** [comment]
Yes, we have one potential `Pro` user asked for this. But I have answered him, we only accept feature request (no 100% promise, but if it makes sense and more and more users ask, we will do for sure) from current `Pro` user, not for potential users. And yes, this feature is for `Pro` only if we do it.

**Reply by Lux-WorldPC** [reply]
I am a Pro user, and this is an option I would need, I am used to it in TeamViewer
for now I have a scheduled task running at windows start up using an admin account

**Reply by OtterKring** [reply]
Same here. I absolutely need the possibility of unattended access, including elevation without confirmation on the client side. We have many "unmanned" machines which need remote control (without using rdp). I am currently evaluation RustDesk versus our existing Teamviewer and this would be a deal-breaker.

**Reply by OtterKring** [reply]
UPDATE: Unattended Access works if I set a fixed password on the client. So far, so good.

**Comment by illheim** [comment]
Second this. I would definitely need unattended access, otherwise I'll have to buy a TeamViewer plan.

**Reply by lionking102** [reply]
I was already integrated into the Pro Version shortly after this post.
